### PR TITLE
Change company transformation to match those for articles and images.

### PR DESCRIPTION
**Pull request by kghatton**
- transform concorded companies to concept element with id and type,
extracting UUID from concordance apiUrl
- use XMLUnit, including in Cucumber tests, for comparing XML fragments
(attribute order is not significant)
- fix a few typos

### CODE REVIEW

File: src/test/java/com/ft/methodearticlemapper/transformation/TearSheetLinksTransformerTest.java [hunk 1]
```diff
@@ -1,184 +1,189 @@
...
+	
+
+	private Document doc;
+	private NodeList nodes;
+	private DocumentBuilder db;
+	private TearSheetLinksTransformer toTest;
+	private Client client = mock(Client.class);
+    private Identifier identifier = new Identifier(TME_AUTHORITY, TME_ID_2);
+    private ConceptView concept= new ConceptView(API_URL2 , API_URL2 );
+    private Concordances concordances;
```
Comment (peteclark-ft) [inline src/test/java/com/ft/methodearticlemapper/transformation/TearSheetLinksTransformerTest.java]: This file has some formatting issues, can you auto-format?

File: src/test/java/com/ft/methodearticlemapper/transformation/TearSheetLinksTransformerTest.java [hunk 2]
```diff
@@ -1,184 +1,189 @@
...
+		nodes = getNodeList(doc);
+		String queryUrl = CONCORDANCE_URL + TME_ID_2;
+		mockConcordanceQueryResponse(queryUrl, concordances);
+		toTest.handle(doc, nodes);
+		String actual = serializeDocument(doc);
+
+		String expectedBody = "<body>" + "<p>Some text</p>"
+				+ "<p><concept type=\"http://www.ft.com/ontology/company/PublicCompany\" id=\"" + ORG_ID
+				+ "\">concorded company name</concept></p>" + "</body>";
+//		assertThat(actual, equalTo(expectedBody));
```
Comment (peteclark-ft) [inline src/test/java/com/ft/methodearticlemapper/transformation/TearSheetLinksTransformerTest.java]: This needed?

**Review by peteclark-ft — APPROVED** [review]
Looks good, bar the formatting comments. Hard to tell what's been changed using this diff, but generally the code looks clean 👍 .

**Review by tosan88 — APPROVED** [review]
As Pete said

**Review by yolandeleungFT — APPROVED** [review]

**Pull request merged**